maas/client/facade.py

```diff
@@ -139,7 +139,8 @@ def machines(origin):
             "allocate": origin.Machines.allocate,
             "get": origin.Machine.read,
             "list": origin.Machines.read,
-            "get_power_parameters": origin.Machines.get_power_parameters,
+            "get_power_parameters_for":
```

Comment:
How does the user access this function? Like:

machine.get_power_parameters_for ? 
[...] returns power parameters?